frontend/src/app/app-routing.module.ts

```diff
@@ -1,12 +1,32 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { AuthRoutes } from './auth/auth.routes';
+import { ProjectListComponent } from './components/project-list/project-list.component';
+import { PageWithoutSidebarComponent } from './components/page-without-sidebar/page-without-sidebar.component';
+import { OrganizationListComponent } from './components/organization-list/organization-list.component';
 import { UserRoutes } from './user/user.routes';
+import { PageWithSidebarComponent } from './components/page-with-sidebar/page-with-sidebar.component';
+import { TasqueBoardComponent } from './components/tasque-board/tasque-board.component';
+import { TasqueTeamComponent } from './components/tasque-team/tasque-team.component';
+import { TasqueProjectSettingsComponent } from './components/tasque-project-settings/tasque-project-settings.component';
 
 const routes: Routes = [
   ...AuthRoutes,
-  ...UserRoutes,
-  { path: '**', redirectTo: 'auth' },
+  { path: '', component: PageWithoutSidebarComponent,
+    children: [
+      { path: 'organizations', component: OrganizationListComponent },
+      { path: 'projects', component: ProjectListComponent, },
+      ...UserRoutes,
+    ]
+  },
+  { path: 'project', component: PageWithSidebarComponent,
+    children: [
+      { path: 'board', component: TasqueBoardComponent },
+      { path: 'team', component: TasqueTeamComponent },
+      { path: 'settings', component: TasqueProjectSettingsComponent },
+    ]
+  },
+  { path: '**', redirectTo: '/auth/login', pathMatch: 'full' },
```

Comment:
`/auth` automatically redirects to `/auth/login` so there's no need to be so explicit